packages/react/package.json

```diff
@@ -39,7 +39,7 @@
     "postinstall": "carbon-telemetry collect --install",
     "start": "yarn test:engines && yarn storybook",
     "storybook": "yarn test:engines && start-storybook -p 3000 -s public/development",
-    "svg-convert": "svgr --ext jsx -d src/icons/components src/icons/svg && prettier --write src/icons/components/index.jsx ",
+    "svg-convert": "svgr --ext jsx -d src/icons/components src/icons/svg && prettier --write src/icons/components/index.jsx  && eslint --fix --ext .jsx src/icons/components",
```

Comment:
add eslint --fix to svg-convert to remove curly braces from the icons that will get flagged by linting.